### PR TITLE
Add IBM readme

### PR DESCRIPTION
**Pull request by skaegi**

### CODE REVIEW

File: ibm/readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#IBM
```
Comment (vdemeester) [inline ibm/readme.md]: `# IBM` :stuck_out_tongue_closed_eyes:

**Review by vdemeester — APPROVED** [review]
:+1:

File: ibm/readme.md (hunk 1 above)
Comment (skaegi) [reply ibm/readme.md]: ty ;)

**Pull request merged**